AMRC-FactoryPlus/cs-serviceclient
# Boilerplated all necessary public interfaces

Created the structure of the library overall. Mainly based upon the existing JS library, but with some C# changes.
I believe I have captured the most important parts, and can start to actually implement the methods.

Some of the methods are already implemented, because it's hard not to.

# Testing

1. Build the library. This provides a .dll file buried in the bin folder
2. Reference this library from another .NET application... The sample app for example!
3. Check to make sure the methods are exposed!
